templates/web/base/admin/user-alerts.html

```diff
@@ -0,0 +1,52 @@
+[% IF user.alerts %]
+<h3>[% loc("User's alerts") %]</h3>
+<form method="POST">
+<input type="hidden" name="token" value="[% csrf_token %]" >
+<input type="hidden" name="update_alerts" value="1" >
+<table>
+<tr>
+    <th>ID</th>
+    <th>Type</th>
+    <th>Parameters</th>
+    <th>Confirmed</th>
+    <th>State</th>
+    <th>Details</th>
+    <th>Enable</th>
+    <th>Disable</th>
+    <th>Delete</th>
+</tr>
+[% FOREACH alert IN user.alerts %]
+<tr>
+    <td>[% alert.id %]</td>
+    <td>[% alert.alert_type.ref %]</td>
+    <td>[% alert.parameter %] [% alert.parameter2 %]</td>
```

Comment:
No need for this if the later column lists them too. (Or that could be fallback for that column if not all cases covered?)